packages/form/src/Field.js

```diff
@@ -23,6 +24,7 @@ const Field = ({
   ...attributes
 }) => {
   let row = false;
+  let { inputId } = attributes;
```

Comment:
I think instead of adding a new prop called `inputId` just alias or use the `attributes.id` off the field props.

Then you can declare a const that is something like the below:
`const inputId = attributes.id || uuid()`

Then make sure to set that for both the `label` and the `input` components.

Reply:
I believe the reason why we use `inputId` for `AvSelect` is because the `react-select` package takes in that prop and puts it on the input rather than the top level component

Reply:
The reason I opted against using a const is that I didn't want to create an ID when label is not needed, if you're happy to generate uuid all the time, I can change it though!

Reply:
@GoPro16 do you still want to go with your suggesstions, happy to do it?

Reply:
Yeah you can make it conditional, my only concern was getting rid of the `inputId` prop and using the default `id` field.

Reply:
I moved away from `inputId` in favour of `id`, let me know if you want to make it conditional or you're happy with the latest commit result?